unime/src/routes/+layout.svelte

```diff
@@ -57,46 +52,74 @@
       goto(`/prompt/${type}`);
     }
   }
+
+  const menuItemCss = 'flex flex-col content-start justify-center mr-4';
```

Comment:
Even if it feels like it might be a good idea to extract reused classes into a JS variable, it is actually discouraged. There's a good [article on reusing styles](https://tailwindcss.com/docs/reusing-styles) that I suggest we should follow. I always try to decide case by case among the three options: 1. just copy & paste for each element, 2. loop over elements, 3. extract component

Reply:
I don't really see where it is discouraged, the bottom example with btn-primary seems to be an almost similar pattern. I can try to make a component of it if you really want

Reply:
I would rather go for option 1. or 2.

Reply:
I think you're confusing two things here: the example with `btn-primary` you're refering to is a custom utility class defined through the `@apply` directive of the framework itself that allows you to group classes that are always used together in a single custom class. Your current solution puts the classes in a standard JavaScript string inside the component itself. We don't have IntelliSense anymore, no more automatic class sorting, "premature abstraction". I really do not see a problem with simply copy & pasting the styles since we are only talking about 4-5 elements (which is considered a valid option by the creators of Tailwind: "duplication like this isn't even a real problem because it's all together in one place"). If you really don't like it, we can also define an inline interface inside the component that defines all dynamic parts for a `DevModeButton` and then loop over them and just style once. Extracting to its own separate Svelte component (option 3.) for such a small button seems overkill to me in this case.

Reply:
Its just annoying when you make changes you have to copy again to all the other ones.

Reply:
> If you really don't like it, we can also define an inline interface inside the component that defines all dynamic parts for a DevModeButton and then loop over them and **just style once**.

Reply:
Ok looped now!